src/control/server/mgmt_drpc.go

```diff
@@ -87,12 +87,17 @@ func (mod *srvModule) handleGetPoolServiceRanks(reqb []byte) ([]byte, error) {
 
 	mod.log.Debugf("handling GetPoolSvcReq: %+v", req)
 
+	resp := new(srvpb.GetPoolSvcResp)
+
 	ps, err := mod.sysdb.FindPoolServiceByUUID(uuid)
 	if err != nil {
-		return nil, err
+		resp.Status = int32(drpc.DaosNonexistant)
+		mod.log.Debugf("GetPoolSvcResp: %+v", resp)
+		return proto.Marshal(resp)
+		// return nil, err
 	}
 
-	resp := new(srvpb.GetPoolSvcResp)
+	// resp := new(srvpb.GetPoolSvcResp)
```

Comment:
Forgot to delete?

Reply:
Yes. If there are any other changes needed in this PR I will remove it. Otherwise I will make the change in the next commit.